app/models/lighthouse_supporting_evidence_attachment.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# Files uploaded as part of a form526 submission that will be sent to Lighthouse upon form submission.
+# inherits from ApplicationRecord
+class LighthouseSupportingEvidenceAttachment < FormAttachment
```

Comment:
Is this a replacement for an EVSS object?

Reply:
It is, yes